crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -731,8 +731,16 @@ async fn resolve_internal(
         let result_ref = import_map.lookup(context, request).await?;
         let result = &*result_ref;
         if !matches!(result, ImportMapResult::NoEntry) {
+            // Prevent alias recursively applied infinitely, i.e
+            // paths: { "*": ["node_modules/*"] }
+            let resolve_op = ResolveOptions {
+                import_map: None,
```

Comment:
This should also prevent transitive aliases, which should be fine.

```
paths: {
  "a": ["b"],
  "b": ["c"]
}

Reply:
ugh. any suggestions?